lib/init/config-initializer.js

```diff
@@ -428,14 +428,17 @@ function askInstallModules(modules, packageJsonExists) {
 
     log.info("The config that you've selected requires the following dependencies:\n");
     log.info(modules.join(" "));
-    return inquirer.prompt([
+    return enquirer.prompt([
         {
             type: "confirm",
             name: "executeInstallation",
             message: "Would you like to install them now with npm?",
-            default: true,
-            when() {
-                return modules.length && packageJsonExists;
+            initial: () => true,
+            skip() {
+                return !(modules.length && packageJsonExists);
+            },
+            result(input) {
+                return this.skip ? null : input;
```

Comment:
This result method is needed since enquirer will even return the initial value when the question is skipped, unlike inquirer.